review/management/commands/seed_jawafdehi.py

```
@@ -0,0 +1,344 @@
+            "sources": 0,
+            "cases": 0,
+            "relationships": 0,
+        }
+
+        try:
+            if opts["sources_only"]:
+                self._seed_sources_catalog()
+            else:
+                self._seed_cases(opts)
+        except jds_client.JdsError as e:
+            raise CommandError(str(e))
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)